src/jobs/BenchmarkJob.jl

```diff
@@ -136,33 +136,28 @@ tmpdir(job::BenchmarkJob) = joinpath(workdir, "tmpresults")
 tmplogdir(job::BenchmarkJob) = joinpath(tmpdir(job), "logs")
 tmpdatadir(job::BenchmarkJob) = joinpath(tmpdir(job), "data")
 
-function retrieve_daily_data!(cfg, date)
+function retrieve_daily_benchmark_data!(cfg, date)
     dailydir = joinpath(reportdir(cfg), "benchmark", "by_date", datedirname(date))
-    if isdir(dailydir)
-        cd(dailydir) do
-            datapath = joinpath(dailydir, "data")
-            try
-                open("data.tar.zst") do io
-                    stream = XzDecompressorStream(io)
-                    Tar.extract(stream, datapath)
-                end
-                datafiles = readdir(datapath)
-                primary_index = findfirst(fname -> endswith(fname, "_primary.minimum.json"), datafiles)
-                if primary_index !== nothing
-                    against = match(r"Commit.+\(https://github.com/([^/)]+/[^/)]+)/commit/(\w+).*\)", read(joinpath(dailydir, "report.md"), String))
-                    (repo::String, commit::String) = against === nothing ? ("", "") : (against[1], against[2])
-                    primary_file = datafiles[primary_index]
-                    results = BenchmarkTools.load(joinpath(datapath, primary_file))[1]
-                    return results, repo, commit
-                end
-            catch err
-                nodelog(cfg, myid(), "encountered error when retrieving daily data";
-                        error=(err, catch_backtrace()))
-            finally
-                isdir(datapath) && rm(datapath, recursive=true)
-            end
-            nothing
+    isdir(dailydir) || return nothing
+
+    datapath = joinpath(dailydir, "data.tar.zst")
```

Comment:
missing "data" folder component of path now?

Reply:
No, since the switch to Tar.jl the `data` folder isn't there anymore, and I rewrote the archives in the NanosoldierReports repository accordingly.